src/index.ts

```diff
@@ -50,3 +51,105 @@ const touchManifest = (root: string): void => {
 
   fs.writeFileSync(manifestPath, JSON.stringify({}, null, 2));
 };
+
+const esbuildMultipleBuilds = async function (
+  esbuildOptions: EsbuildOptions,
+): Promise<BuildResult<EsbuildOptions> | void> {
+  const builds = splitEsbuildOptions(esbuildOptions);
+
+  for (const build of builds) {
+    await esbuild.build(build).catch(errorHandler);
+  }
```

Comment:
This actually won't run in parallel as you're expecting. This will run builds one by one. 

If you truly wanted parallel builds you could do this:

```suggestion
await Promise.allSettled(
  builds.map(async (build) => await esbuild.build(build).catch(errorHandler))
)
```

Reply:
I tried this, it is fast but it didn't seem to work with the manifest file and it being valid JSON all the time.

Reply:
In this case, I think we'd need to add a mutex around the part that updates the manifest file.